### ISSUE TITLE
Issues with line charts

### ISSUE THREAD

**Issue by MrichardsPSRC** [psrc/psrcplot#33]
I am attempting to create a line chart to show data over time. I have a test data set where the year is double: 
![image](https://user-images.githubusercontent.com/72169299/213818892-b52a5a55-f997-456a-937f-b46617240de7.png)

When I try to run the `interactive_line_chart` function, I get an error message
![image](https://user-images.githubusercontent.com/72169299/213818790-337197e3-0cf0-4d35-9423-437464a1e8e5.png)

So, I try to convert the year field into date format:
```
# need to make the year column date format
lifeexpec_poc_trend$year_date <- as.Date((as.character(lifeexpec_poc_trend$year)),
                                         format = "%Y") 
```
When I run the script again, the chart object is null? 
![image](https://user-images.githubusercontent.com/72169299/213819707-c52e8cd1-0f54-4b17-8eef-b458b9653ab3.png)
 

![image](https://user-images.githubusercontent.com/72169299/213819790-50c0f534-6b2e-435d-a71d-587c9e1973c8.png)

The `static_line_chart` function did work, to an extent:
```
line_option <- static_line_chart(t=lifeexpec_poc_trend,
                                 x="year_date", y="health_ind",
                                 fill="quintile_order",
                                 est = "number",
                                 color = "blues_inc",
                                 title = indicator_measure,
                                 subtitle = "People of Color",
                                 source = paste("Sources: Washington Tracking Network, Washington State Department of Health,", "\n", "U.S. Census Bureau, PUMS 1-year, 2015-2020,", "\n", "2020 Census Tracts"))
```
![image](https://user-images.githubusercontent.com/72169299/213821399-964e22e0-78a9-47c2-9a40-859d80996bbd.png)

**Comment by MrichardsPSRC** [comment]
I also tried `generic_line` (https://psrc.github.io/psrcplot/reference/generic_line.html), but was getting an error message:
![image](https://user-images.githubusercontent.com/72169299/213820328-17538569-592c-4483-8938-71d177c1d79d.png)

I'm not sure if this is actually a function that we can use? It's listed on the documentation page, but I'm not sure what the difference between this function and the `static_line_chart` and `interactive_line_chart`

**Comment by jensenmj** [comment]
NULL object fixed with commit 843c0af. 
BTW character date can be combined with xtype="Continuous" (default is "Date"), or if Date, year format improved with dform="%Y". It may be helpful for the function to read the type rather than rely on an argument.
